frontend/src/lib/constants.tsx

```diff
@@ -147,8 +147,10 @@ export enum FunnelLayout {
 export const BIN_COUNT_AUTO = 'auto'
 
 // Cohort types
-export const COHORT_STATIC = 'static'
-export const COHORT_DYNAMIC = 'dynamic'
+export enum CohortTypeType {
```

Comment:
Nit: can we name this something slighly different? Maybe `CohortTypeEnum`. Just hard to read

Reply:
Idk CohortTypeType kind of rolls off the tongue /s

Reply:
😂